test/dotnet.Tests/CommandTests/Project/Convert/DotnetProjectConvertTests.cs

```diff
@@ -810,7 +810,7 @@ public void Directives_Empty()
 
     [Theory, CombinatorialData]
     public void Directives_EmptyName(
-        [CombinatorialValues("sdk", "property", "package")] string directive,
+        [CombinatorialValues("sdk", "property", "package", "project")] string directive,
```

Comment:
A virtual project will still be produced in this case right? I guess we also do that for the empty name cases for the other directives listed here.

Reply:
In case of `run-api`, yes, that one collects all errors and produces the virtual project file (without directives that have errors), there are tests for that in general (`Api_Diagnostic_*`), albeit not this particular case. In case of other commands like `run`, the first error makes the whole command fail, so there is no virtual project produced.